src/class-wpml-elementor-data-settings.php

```diff
@@ -34,8 +34,14 @@ public function add_hooks() {
 	 * @return mixed
 	 */
 	public function mark_css_field_as_empty( $value, $translated_post_id, $original_post_id, $meta_key ) {
-		if ( '_elementor_css' === $meta_key ) {
-			$value['status'] = '';
+		if ( '_elementor_css' === $meta_key && is_array( $value ) ) {
+			if ( ! isset( $value['status'] ) ) {
+				$value           = current( $value );
+				$value['status'] = '';
+				$value           = array( $value );
+			} else {
+				$value['status'] = '';
+			}
```

Comment:
@amElnagdy how this part is related to the scope of the PR?

Also, this part is very confusing:

```php
$value           = current( $value );
$value['status'] = '';
$value           = array( $value );
```

Try to not reuse the same variable for the same purpose.
Use new variables instead, and give them a meaningful name, so the code will be understandable.

Something like that (but with better names, please):
```php
$current_value           = current( $value );
$current_value['status'] = '';
$value                   = array( $current_value );
```

I can't suggest any naming, because I really don't know why you are doing this, why you are setting the "status" key only to the "current" item of the `$value` array and then basically replace `$value` with a new array made of only one item.

Reply:
@andreasciamanna Adriano authored this and I have no clue how this got into this PR.

Reply:
@amElnagdy I think it might get solved if you rebase the branch.